buildSrc/src/main/java/org/elasticsearch/gradle/test/rest/CopyRestApiTask.java

```diff
@@ -58,8 +58,10 @@
     final ListProperty<String> includeCore = getProject().getObjects().listProperty(String.class);
     final ListProperty<String> includeXpack = getProject().getObjects().listProperty(String.class);
     String sourceSetName;
+    boolean skipHasRestTestCheck;
     Configuration coreConfig;
     Configuration xpackConfig;
+    Configuration additionalConfig;
```

Comment:
This is needed as a way to copy the 7.x module/plugin tests to the source set. For "normal" YAML tests they are already in the source set (since they are part of the current source code).